src/mod/applications/mod_dptools/mod_dptools.c

```diff
@@ -1581,14 +1581,14 @@ SWITCH_STANDARD_APP(video_refresh_function)
 		} else if (!strcasecmp(cmd, "auto")) {
 			switch_channel_clear_flag(channel, CF_MANUAL_VID_REFRESH);
 		}
-		
-		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, 
-						  "%s video refresh now in %s mode.\n", switch_channel_get_name(channel), 
-						  switch_channel_test_flag(channel, CF_MANUAL_VID_REFRESH) ? "manual" : "auto");	
+
+		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG,
+						  "%s video refresh now in %s mode.\n", switch_channel_get_name(channel),
+						  switch_channel_test_flag(channel, CF_MANUAL_VID_REFRESH) ? "manual" : "auto");
 
 		return;
 	}
-	
```

Comment:
Don't fix the tab indention on this commit. We are merging to 1.10.5 soon so we might need to merge our changes once its done. So keep the commit simple for now

Reply:
Got it, fixed it. :)